controller.rb

```diff
@@ -0,0 +1,37 @@
+require 'pry'
+require_relative 'Parser'
+require_relative 'deck'
+require_relative 'view'
+
+class Controller
+  # include Parser
+  attr_reader :viewer, :deck
+
+  def initialize(file)
+    @parsed_file = Parser.read(file)
+    @viewer = View.new
+    @deck = Deck.new(@parsed_file)
+    @deck.generate_random_card
+    @viewer.welcome_message
+    run_flashcard_game
+  end
+
+  def run_flashcard_game
+    @viewer.show_definition(@deck.flashcards.first)
+    player_input = @viewer.input
+    if player_input == "exit"
+      return @viewer.goodbye_message
+    elsif player_input == @deck.flashcards.first.term
```

Comment:
Again, I think this is a job for the Deck:

`@deck.correct_answer_for_current_card?(player_input)`